plugin-dev/Sentry.uplugin

```diff
@@ -31,7 +31,7 @@
 			"sh $(PluginDir)/Scripts/upload-debug-symbols.sh $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
 		],
 		"Win64": [
-			"powershell -ExecutionPolicy RemoteSigned -File $(PluginDir)/Scripts/upload-debug-symbols-win.ps1 $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
+			"pwsh -ExecutionPolicy bypass -File $(PluginDir)/Scripts/upload-debug-symbols-win.ps1 $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
```

Comment:
Can we stick to `powershell` which should be installed by default?

same above in the `ps1`

Reply:
It wasn't running on the base version of PowerShell, I can't find the thread right now, but @tustanivsky had similar problems with the default powershell version.

Reply:
I recall having issues with setting `SENTRY_PROPERTIES` env variable when using the default PowerShell version, though I wasn't able to reproduce it lately.

Reply:
I remember that, but there have been changes in the meantime. Let's see if that helps (in the next release) and get back to this if it doesn't.